### PR TITLE
add W->3MuNu cards at 13.6 TeV

### PR DESCRIPTION
**Pull request by lguzzi**
This PR adds proc and run cards for the generation of pp->W+->mu+mu+mu- nu (and cc) at 13.6 TeV within the standard model.
This is required to produce a background sample for the tau->3mu analysis

### CODE REVIEW

File: bin/MadGraph5_aMCatNLO/cards/ppW3MuNu_smfull/ppW3MuNu_smfull_proc_card.dat [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+set gauge unitary
+set complex_mass_scheme False
+set max_npoint_for_channel 0
+import model sm
+define p = g u c d s u~ c~ d~ s~
+define j = g u c d s u~ c~ d~ s~
+define l+ = e+ mu+
+define l- = e- mu-
+define vl = ve vm vt
+define vl~ = ve~ vm~ vt~
```
Comment (menglu21) [inline bin/MadGraph5_aMCatNLO/cards/ppW3MuNu_smfull/ppW3MuNu_smfull_proc_card.dat:37]: the banner can be removed

File: bin/MadGraph5_aMCatNLO/cards/ppW3MuNu_smfull/ppW3MuNu_smfull_run_card.dat [hunk 2]
```diff
@@ -0,0 +1,270 @@
...
+#                                                                    
+#*********************************************************************
+# Tag name for the run (one word)                                    *
+#*********************************************************************
+  tag_1     = run_tag ! name of the run 
+#*********************************************************************
+# Number of events and rnd seed                                      *
+# Warning: Do not generate more than 1M events in a single run       *
+#*********************************************************************
+  1000000 = nevents ! Number of unweighted events requested
```
Comment (menglu21) [inline bin/MadGraph5_aMCatNLO/cards/ppW3MuNu_smfull/ppW3MuNu_smfull_run_card.dat:30]: change 100000 to 5000

File: bin/MadGraph5_aMCatNLO/cards/ppW3MuNu_smfull/ppW3MuNu_smfull_run_card.dat [hunk 3]
```diff
@@ -0,0 +1,270 @@
...
+     1        = lpp1    ! beam 1 type 
+     1        = lpp2    ! beam 2 type
+     6800.0     = ebeam1  ! beam 1 total energy in GeV
+     6800.0     = ebeam2  ! beam 2 total energy in GeV
+# To see polarised beam options: type "update beam_pol"
+#*********************************************************************
+# PDF CHOICE: this automatically fixes also alpha_s and its evol.    *
+#*********************************************************************
+     nn23lo1    = pdlabel     ! PDF set                                     
+     230000    = lhaid     ! if pdlabel=lhapdf, this is the lhapdf number
```
Comment (menglu21) [inline bin/MadGraph5_aMCatNLO/cards/ppW3MuNu_smfull/ppW3MuNu_smfull_run_card.dat:46]: change these two lines to:
  'lhapdf'    = pdlabel     ! PDF set                                     
     $DEFAULT_PDF_SETS    = lhaid     ! if pdlabel=lhapdf, this is the lhapdf number
     $DEFAULT_PDF_MEMBERS = reweight_PDF

File: bin/MadGraph5_aMCatNLO/cards/ppW3MuNu_smfull/ppW3MuNu_smfull_run_card.dat [hunk 4]
```diff
@@ -0,0 +1,270 @@
...
+#
+#*********************************************************************
+# Store info for systematics studies                                 *
+# WARNING: Do not use for interference type of computation           *
+#*********************************************************************
+   True  = use_syst      ! Enable systematics studies
+#
+systematics = systematics_program ! none, systematics [python], SysCalc [depreceted, C++]
+['--mur=0.5,1,2', '--muf=0.5,1,2', '--pdf=errorset'] = systematics_arguments ! see: https://cp3.irmp.ucl.ac.be/projects/madgraph/wiki/Systematics#Systematicspythonmodule
+# Syscalc is deprecated but to see the associate options type'update syscalc'
```
Comment (menglu21) [inline bin/MadGraph5_aMCatNLO/cards/ppW3MuNu_smfull/ppW3MuNu_smfull_run_card.dat:270]: remove those lines

**Comment by lguzzi** [comment]
@menglu21 , thank you for the review, I updated the cards

File: bin/MadGraph5_aMCatNLO/cards/ppW3MuNu_smfull/ppW3MuNu_smfull_proc_card.dat [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
```
Comment (menglu21) [inline bin/MadGraph5_aMCatNLO/cards/ppW3MuNu_smfull/ppW3MuNu_smfull_proc_card.dat:15]: they are not needed

**Review by menglu21 — COMMENTED** [review]
btw, you should move those cards to MadGraph5_aMCatNLO/cards/production/13p6TeV since it's run3 cards

**Pull request merged**